src/actions/userActions.js

```diff
@@ -0,0 +1,33 @@
+import { SubmissionError } from 'redux-form/immutable';
+
+import userApi from '../api/userApi';
+import authUtils from '../utils/authUtils';
+import * as types from './actionTypes';
+
+export const loginSuccess = () => {
+  return {
+    type: types.LOGIN_SUCCESS
+  };
+};
+
+export const login = (user) => {
+  return (dispatch) => {
```

Comment:
Yeah, you are right. I'm going to make the changes to remove the curly braces where there is one statement. I sometimes use it like this for readability's sake.

However, for the cases when there is only one parameter, I know you can remove the parenthesis, but if the body is between curly braces then there is an eslint rule (arrow-parens) that throws me an error, and since I'm using the eslint file that was created in Rootstrap I'm not sure if it would be better to remove the rule or just leave it as is. What do you think?

Reply:
Sorry I meant that you can remove the parenthesis on the left but only on the cases where you use parenthesis on the right as is the case of this line.

Reply:
Ok! Yeah, I removed it in the last commit. 🙂